main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+# Purses and Payments
+
+You can store `amount` objects in two kinds of objects:
+- **[`purse`](https://agoric.com/documentation/glossary/#purse)**: Hold
+  an `amount` of same branded digital assets until you withdraw part or
+  all of them into a `payment` for use.
+- **[`payment`](https://agoric.com/documentation/glossary/#purse)**:
+  Hold an `amount` of digital assets to send to another party. Usually either
+  a freshly created `amount` or one taken from a `purse`
+
+Assets in purses and payments do not have to be currency-like, but can
+be anything you can construct a digital rights `amount` object for;
+swords to use in a game, rights to use a particular contract, theater
+tickets, etc.
+
+Each non-empty `purse` and `payment` object contains exactly one
+`amount` object. While you may deposit or withdraw an `amount` to a
+`purse`, it is repsectively added to the `purse` `amount`or split off into a `payment`
+object's `amount`. The same is true for adding to or splitting a
+`payment` object's `amount`.  Both a `purse` and a `payment` can only
+have an amount of a single `brand`.
+
+With the exception of `mint` objects creating entirely new `payment`
+objects with newly created `amount` objects, a `payment` object is
```

Comment:
No, there is no such thing as amount objects and they are not created when minted. The digital assets are created when minted, and the digital assets are of a certain amount.
```suggestion
objects with newly created `amount` objects, a `payment` object is
```